library/std/src/sys/unix/weak.rs

```diff
@@ -43,7 +43,7 @@ pub(crate) macro weak {
     )
 }
 
-#[cfg(all(not(any(target_os = "macos", target_os = "ios")), bootstrap))]
+#[cfg(not(any(target_os = "macos", target_os = "ios")))]
 pub(crate) macro weak {
```

Comment:
This whole item should go away with `bootstrap`, in favor of the `not(bootstrap)` one above.